<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>928</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Visualisierung 1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="controls" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QProgressBar" name="progressBar">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelTop">
         <property name="text">
          <string>No data loaded</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="glLayout"/>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Camera</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Projection</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_FieldOfView">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Field Of View</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>90</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Near Plane</string>
          </property>
         </widget>
         <widget class="QComboBox" name="switch_projection">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>30</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>Perspective</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Orthographic</string>
           </property>
          </item>
         </widget>
         <widget class="QSlider" name="slider_FieldOfView">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>60</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>179</number>
          </property>
          <property name="value">
           <number>45</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="box_nearPlane">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>90</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
          <property name="minimum">
           <double>0.010000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.100000000000000</double>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>120</y>
            <width>51</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Far Plane</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="box_FarPlane">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>120</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
          <property name="maximum">
           <double>50000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>500.000000000000000</double>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Rendering</string>
         </property>
         <widget class="QComboBox" name="switch_renderingMode">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>30</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>Front Faces</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Back Faces</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MIP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Alpha-Compositing</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_renderingMode">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>47</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Mode</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>GPU</string>
         </property>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Used Memory (MB)</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="usedMemLCD">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>60</y>
            <width>64</width>
            <height>23</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>51</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Total Memory (MB)</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="memSizeLCD">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>20</y>
            <width>64</width>
            <height>23</height>
           </rect>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Filled</enum>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1183</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open ...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
